server/src/packages/core/index.spec.ts

```diff
@@ -0,0 +1,16 @@
+// Test that uuids are correctly evaluated;
+import { v4 } from 'uuid';
+import { Uuid, UUIDRefinement, uuidCheck} from '.';
```

Comment:
`UUIDRefinement` and `auuidCheck` should be getting caught as unused imports by the linter? Have we disabled this in the server side linting?

Reply:
Possibly, I'll re-enable it in another PR